keps/sig-network/2091-cluster-network-policy/README.md

```diff
@@ -93,40 +93,52 @@ to begin with? AdminNetworkPolicy is a popular alternative.
 
 ## Motivation
 
-Kubernetes provides NetworkPolicy resources to control traffic within a
-cluster. NetworkPolicies focus on expressing a developers intent to secure
-their applications. Thus, in order to satisfy the needs of a security admin,
-we propose to introduce a new API that captures the administrators intent.
+Kubernetes provides the NetworkPolicy resource to control traffic within a
+cluster. NetworkPolicy focuses on expressing a developer's intent to secure
+their applications. However, it was not intended to be used for administrative
```

Comment:
I think "for Cluster scoped administrative traffic control" would be a bit better here.